### PR TITLE
fix(EDA-93): Video file that is larger than 1 GB 

### PR DESCRIPTION
**Pull request by shandev18**
If a video size is larger than 1GB ,we display proper appropriate error message

### CODE REVIEW

File: src/containers/analyze/upload/components/upload.tsx [hunk 1]
```diff
@@ -7,29 +7,39 @@ import {
...
+    accept: ".mp4,.mov,.avi,.wmc,.avchd,.flv,.webm,.mkv,.mpeg,.vob,.wmv",
+    beforeUpload: (file) => {
+      if (file.size > MAX_FILE_SIZE) {
+        message.error(
+          `This file is too large. File size must be less than 1GB.`
+        );
+        return Upload.LIST_IGNORE; // Prevent the file from being uploaded
+      }
+      setTimeout(() => {
+        // Code to hide the error message
```
Comment (xiarahman) [inline src/containers/analyze/upload/components/upload.tsx:32]: is this a placeholder?

File: src/containers/analyze/upload/components/upload.tsx [hunk 2]
```diff
@@ -7,29 +7,39 @@ import {
...
+    },
     onChange(info) {
       const { status } = info.file;
 
       if (status === "done") {
         push(`/result/video/${info.file.response.job_id}`);
       } else if (status === "error") {
-        message.error(
-          `${info.file.name} file upload failed. Supported video formats: mp4 ,mov ,avi ,wmc ,avchd ,flv ,webm ,mkv ,mpeg ,vob ,wmv`
-        );
```
Comment (xiarahman) [inline src/containers/analyze/upload/components/upload.tsx:32]: So we are not showing the error of invalid file?

File: src/containers/analyze/upload/components/upload.tsx [hunk 3]
```diff
@@ -7,29 +7,39 @@ import {
...
+        message.error(` File upload failed. `);
```
Comment (xiarahman) [inline src/containers/analyze/upload/components/upload.tsx:42]: Why?

File: src/containers/analyze/upload/components/upload.tsx [hunk 4]
```diff
@@ -7,29 +7,39 @@ import {
...
+  const MAX_FILE_SIZE = 1 * 1024 * 1024 * 1024; // 1GB in bytes
 
   const props: UploadProps = {
     name: "video",
     multiple: false,
     method: "post",
     action: `${API_ENDPOINT}/analyze_video`,
     listType: "picture",
-    accept: ".mp4,.mov,.avi,.wmc,.avchd,.flv,.webm,.mkv,.mpeg,.vob,.wmv", // Adding the accept attribute here
+    accept: ".mp4,.mov,.avi,.wmc,.avchd,.flv,.webm,.mkv,.mpeg,.vob,.wmv",
```
Comment (xiarahman) [inline src/containers/analyze/upload/components/upload.tsx:23]: What's removed? Comments? Comments should be everywhere.

File: src/containers/analyze/upload/components/upload.tsx (hunk 2 above)
Comment (shandev18) [reply src/containers/analyze/upload/components/upload.tsx:32]: We are now displaying the error messages that come from the backend, avoiding hard-coded error messages from the frontend.

File: src/containers/analyze/upload/components/upload.tsx (hunk 3 above)
Comment (shandev18) [reply src/containers/analyze/upload/components/upload.tsx:42]: We now display the supported file formats on the uploader before the user uploads a file. Additionally, we have removed the file name from the error message, as it is unnecessary and makes the message too long to read.

File: src/containers/analyze/upload/components/upload.tsx (hunk 1 above)
Comment (shandev18) [reply src/containers/analyze/upload/components/upload.tsx:32]: Yes, it is a placeholder for a setTimeout function. It includes a comment indicating where the actual code to hide an error message should be placed but for now not used for that purpose. we use the setTimeout function to set and execute after 15,000 milliseconds or 15 seconds

File: src/containers/analyze/upload/components/upload.tsx (hunk 4 above)
Comment (shandev18) [reply src/containers/analyze/upload/components/upload.tsx]: comment added again and now the formats are handled globally

**Review by xiarahman — APPROVED** [review]
LGTM

**Pull request merged**